ui/src/VoxettaUserForm.js

```diff
@@ -103,11 +111,7 @@ export class VoxettaUserForm extends LitElement {
 
     constructor() {
         super();
-        this.cookieService = new CookieService();
-        this.userId = this.cookieService.getCookieValue("userId");
-        this.gender = this.cookieService.getCookieValue("gender");
-        this.userAge = this.cookieService.getCookieValue("userAge");
-        this.deviceType = this.cookieService.getCookieValue("deviceType"); 
+        this.addEventListener('input', this.formIsValid);
```

Comment:
I think `mwc-select` fires `@selected` not `@input`, so this wouldn't catch changes to gender.

Reply:
Oh good catch! I just tried using 'onselect' and 'onchange' but it doesn't seem to recognize either--I'm getting errors both creating an independent event & putting it within the HTML--I ultimately used an additional independent 'onclick' event (I figured the user will have to click to make a selection) but this is less ideal because it makes some unnecessary checks. I will revisit in the future to see if I can get an 'onchange' variation working